cli/src/main/java/org/opentosca/toscana/cli/commands/platform/PlatformInfo.java

```diff
@@ -13,7 +10,7 @@
     description = {"Information about the specified Platform"},
     customSynopsis = "@|bold toscana platform info|@ @|yellow -p=<name>|@ [@|yellow -mv|@]%n",
     optionListHeading = "%nOptions:%n")
-public class PlatformInfo extends AbstractCommand implements Runnable {
+public class PlatformInfo extends AbstractCommand {
 
     @Option(names = {"-p", "--platform"}, required = true, paramLabel = Constants.PARAM_PLATFORM, description = "Information about the Platform")
     private String platformInfo;
```

Comment:
Does this contain the value of the flag (the platform id)? The name does not really tell you what this stores